city_scrapers/spiders/bisnd_bps.py

```diff
@@ -0,0 +1,158 @@
+import html
+import json
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import BOARD, CANCELLED, COMMITTEE, PASSED, TENTATIVE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from lxml import html as lhtml
+from scrapy import Selector
+
+
+class BisndBpsSpider(CityScrapersSpider):
+    name = "bisnd_bps"
+    agency = "Bismarck Public Schools"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.bismarckschools.org/Page/401"]
+
+    """
+    Meetings' time and location is taken from organization's following webpage:
+    https://www.bismarckschools.org/Page/398
+    """
+    meeting_time = time(17, 15)
+    location = {
+        "name": "Tom Baker Meeting Room of the City/County Office Building",
+        "address": "221 N Fifth Street, Bismarck, ND",
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+
+        extracted_input = response.css(
+            '.ui-widget-detail input[type="hidden"]::attr(value)'
+        ).extract_first()
+
+        parsable_data = self._parsed_data(extracted_input)
+
+        for item in parsable_data:
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
```

Comment:
**polish (non-blocking):** Similar to above. Not a big deal. Since we're not really generating a title here, I'd suggest simple re-writing this docstring as "Parse meeting title" or removing the docstring entirely and letting the method name speak for itself.